Gem/Code/Source/RHI/InputAssemblyExampleComponent.cpp

```diff
@@ -259,7 +259,7 @@ namespace AtomSampleViewer
                 frameGraph.UseShaderAttachment(attachmentDescriptor, RHI::ScopeAttachmentAccess::ReadWrite);
             }
 
-            frameGraph.SetEstimatedItemCount(1);
+            frameGraph.SetEstimatedItemCount(2);
```

Comment:
Why does the the ItemCount need to change here?

Reply:
It's doing multiple submits.  The previous item count of 1 was therefore incorrect.